README.md

```diff
@@ -14,47 +14,56 @@ substrate-api-client a library written in Rust for connecting to the substrate's
 ## Prerequisites
 
 In order to build the substrate-api-client and the examples, Rust and the wasm target are needed. For Linux:
-
-    curl https://sh.rustup.rs -sSf | sh
-
-    rustup default nightly
-    rustup target add wasm32-unknown-unknown --toolchain nightly
-
+```
+curl https://sh.rustup.rs -sSf | sh
+rustup show
+```
 For more information, please refer to the [substrate](https://github.com/paritytech/substrate) repository.
 
 ## Substrate node
 
-To execute the examples, a running substrate node is needed. You can download our node artifact from our GitHub Actions
-CI, which can be found in the 'Actions' tab (to the right of the Pull requests tab).
-
-The unpacked node can be run with:
+To execute the examples, a running substrate node is needed. You can download a node artifact from substrate directly: https://github.com/paritytech/substrate
+or run the kitchensink-node with docker:
 
 ```
-./node-template --dev
+docker run -p 9944:9944 -p 9933:9933 -p 30333:30333 parity/substrate:latest --dev --ws-external --rpc-external
 ```
 
 ## Examples
 
 To run an example, clone the `substrate-api-client` repository and run the desired example directly with the cargo command:
 
 ```bash
-    git clone https://github.com/scs/substrate-api-client.git
-    cd substrate-api-client
-    cargo run --example example_get_storage
+git clone https://github.com/scs/substrate-api-client.git
+cd substrate-api-client
+cargo run --example example_get_storage
 ```
+or download the already built binaries from Github Actions: https://github.com/scs/substrate-api-client/actions and run them without any building:
 
-Set the output verbosity by prepending `RUST_LOG=info` or `RUST_LOG=debug`.
+```bash
+chmod +x EXAMPLE
+./EXAMPLE
```

Comment:
Is `EXAMPLE` a placeholder for the example of choice? If yes, you usually use brackets for placeholders `<example>` or `<EXAMPLE>`.

Reply:
Good input, fixed in [6e71aaa](https://github.com/scs/substrate-api-client/pull/305/commits/6e71aaa2732fefefa84e3d199813fcdf53cd26ae)